README.md

```diff
@@ -74,33 +74,39 @@ You'll need a locally running Docker and Docker Compose:
 
 ### Quickstart
 
+Modify or create `.env.local`. This config uses the database container defaults.
+```
+DATABASE_URL=mysql://root:@database:3306/wsexport?serverVersion=5.7
+```
+
+Make sure you cd into `./docker`
+
+```bash
+cd ./docker 
```

Comment:
Starting from scratch, this directory doesn't exist yet

Reply:
there files in there. It should exists. I'm very confused

Reply:
I think I can see how instructions are confusing. Maybe this can clear it a little bit?

1. create/modify `./.env.local`. This is your symfony local env override and you need to setup your db connection and whatever else you want to override regarding symfony and the app itself
2. cd into `./docker`
3. ... everything else (docker related commands)

Reply:
Sorry! I misread this as `.docker` and not `./docker` :) I think the instructions are fine